setup.py

```diff
@@ -171,10 +171,10 @@ def find_data_files(package_dir, patterns):
     keywords='machine learning control physics MuJoCo AI',
     install_requires=[
         'absl-py>=0.7.0',
-        'enum34',
+        'enum34; python_version < "3.4"',
         'dm_env',
         'future',
-        'futures',
+        'futures; python_version == "2.7"',
```

Comment:
`python_version < "3.2"` seems better, since `futures` was added in 3.2

Reply:
According to the package readme:
> It does not work on Python 3 due to Python 2 syntax being used in the codebase.

Reply:
Ah I see, thanks for clarifying